core-web/libs/dotcms-scss/angular/dotcms-theme/components/_table.scss

```diff
@@ -8,25 +13,278 @@ p-table {
             border-radius: $border-radius-md;
         }
 
-        .p-datatable-thead {
-            > tr > th {
-                background-color: $color-palette-gray-200;
-                color: $black;
+        & .p-paginator-top,
+        & .p-paginator-bottom {
+            border-width: 0 0 1px 0;
+            border-radius: 0;
+        }
+
+        & .p-datatable-header,
+        & .p-datatable-footer {
+            background: $white;
+            color: $black;
+            border: 1px solid $table-border-color;
+            border-width: 0 0 1px 0;
+            padding: $spacing-2 $spacing-2;
+            font-weight: $font-weight-semi-bold;
+        }
+
+        & .p-datatable-thead > tr > th {
+            background-color: $color-palette-gray-200;
+            border: 1px solid $color-palette-gray-300;
+            border-width: 0 0 1px 0;
+            color: $black;
+            font-size: $font-size-xsm;
+            font-weight: $font-weight-semi-bold;
+            padding: $spacing-1;
+            text-align: left;
+            transition: none;
+            text-wrap: nowrap;
+
+            &:first-child {
+                padding-left: $spacing-3;
+            }
+
+            &:last-child {
+                padding-right: $spacing-3;
             }
         }
 
-        .p-datatable-tbody {
-            > tr:hover {
+        & .p-datatable-tfoot > tr > td {
+            text-align: left;
+            padding: $spacing-2 $spacing-2;
+            border: 1px solid $table-border-color;
+            border-width: 0 0 1px 0;
+            font-weight: $font-weight-semi-bold;
+            color: $black;
+            background: $white;
+        }
+
+        & .p-sortable-column {
+            outline: 0 none;
+
+            & .p-sortable-column-icon {
+                display: inline-block;
+                color: $color-palette-gray-700;
+                margin-left: $spacing-1;
+            }
+
+            & .p-sortable-column-badge {
+                border-radius: 50%;
+                height: 1.143rem;
+                min-width: 1.143rem;
+                line-height: 1.143rem;
```

Comment:
not really, this is for the badge in case we use multiple sort + a custom sort icon: 
<img width="234" alt="image" src="https://github.com/dotCMS/core/assets/31667212/40e7ca37-397d-4f0d-8cc2-899d626e2f5d">